src/icons/annotations/IconRegionAnnotation.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { white } from '../../styles/variables';
+
+import AccessibleSVG from '../accessible-svg';
+
+import { Icon } from '../iconTypes';
+
+const IconRegionAnnotation = ({ className = '', color = white, height = 21, title, width = 22 }: Icon) => (
```

Comment:
Are these dimensions correct? Looks like preview SDK has different dimensions

Reply:
It looked a little small with the smaller dimensions